src/App.jsx

```diff
@@ -14,45 +19,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function sortGoodsBy(goodList, sortBy, isNeedReverse) {
+  const goodListCopy = [...goodList];
+
+  goodListCopy.sort((good1, good2) => {
+    switch (sortBy) {
+      case ('alpabetic'):
```

Comment:
you already have constants for this cases
```suggestion
      case SORT_BY_ALPHABETIC:
```